src/Contact/View/page.twig

```diff
@@ -1,23 +1,30 @@
 {% include 'Common/View/header.twig' %}
-<section class="container">
-        <form method ="post" action="">   
-                <h1>NAPISZ DO NAS</h1>
+<section class="container contact-section">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css">
```

Comment:
You had good intuition that styles for our page should be adding in <link> element on the beginign of html file :) This fragment of code already exist in our project and live inside of <head> element at the begining of entire html file. In our project you can find this place in src/Common/Header.twig. There is the beginign of our html. So, you can delete this line from here.